hmf-common/src/main/java/com/hartwig/hmftools/common/isofox/GeneExpressionDistributionData.java

```diff
@@ -109,4 +110,14 @@ public double getTpmPercentile(final String geneId, final String cancerType, fin
 
         return getPercentile(percentileMap.get(cancerType), sampleTpm);
     }
+
+    Set<String> configuredCancerTypes()
```

Comment:
NotNull?

Reply:
I considered it but there are no principles around this usage in hmftools so I usually keep the file I work in consistent (and this file didn't use nullable annotations)